static/src/javascripts/projects/common/modules/identity/wizard.js

```diff
@@ -22,6 +22,25 @@ const stepTransitionClassnames = [
     stepOutReverseClassname,
 ];
 
+const getIdentifier = (wizardEl: HTMLElement) =>
+    fastdom.read(() => (wizardEl.id ? wizardEl.id : containerClassname));
```

Comment:
@gustavpursche i'm not _thrilled_ about this because if one page has 2+ wizards with no ID (which, i mean, would be kind of a weird layout to have, but still) they'll all pick up history events. 

Can't really think of a solution other than assigning uuids to them which seems overkill unless we already have a lib in place.

Reply:
it's a nicer syntax but doesn't really address the underlying issue :(

Reply:
Yeah I know, but let's keep it for now. As long as there is no case of 2 wizards on the same page, why invent a super complex solution to a non existing problem :)